app/grandchallenge/evaluation/models.py

```diff
@@ -1720,6 +1770,9 @@ class Evaluation(CIVForObjectMixin, ComponentJob):
     ground_truth = models.ForeignKey(
         EvaluationGroundTruth, null=True, blank=True, on_delete=models.PROTECT
     )
+    inputs = models.ForeignKey(
```

Comment:
I think that overriding the field name is dangerous. When interacting with the base class `inputs` will return two separate things depending on the concrete model, one returns an object, the other a queryset, right?

The quickest fix here might be to give it a different name, but then we would need to make sure that `evaluation.inputs` is never used.

I think that a complete implementation would be to migrate other concrete classes of `ComponentJob` too.

Reply:
I share your concerns, but that is why the migrations is a multi step process. I first add a new field with a different name (input_set), then migrate the inputs, then remove the old `inputs` field and then rename the new field from `input_set` to `inputs`. I tried this out, and from what I can tell it works fine. 

But I think we both agree that only migrating the inputs for the `Evaluation` anyway is suboptimal

Reply:
Agreed, thank for trying!